scripts/test-setup.sh

```diff
@@ -12,6 +12,23 @@ CONT_NAME="${OS_TYPE}-${OS_VERSION}-${COMPONENT}"
 CERTGEN_REPO="https://github.com/redhat-qe-security/certgen"
 CERTGEN_PATH="openssl/Library/certgen"
 
+# Test sanity check
+# Check if all tests have rlGetTestState at their end
+FAILED_CHECKS=0
+FAILED_NAMES=()
+while read file; do
+    if ! grep -Pzq "rlGetTestState[[:space:]]*$" "$file"; then
```

Comment:
Hm, that's interesting, I've never seen this error on any distro I tested this on. However, it returns 0 even in case when it should fail (on RHEL/CentOS 6.8+ and 7.3+):

E.g.:
```
# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 6.8 (Santiago)
# echo -e "rlGetTestState\nrlRun 'echo'" | grep -Pzq 'rlGetTestState[[:space:]]*$'; echo $?
0
```

Nevertheless, it can be easily fixed by replacing the `$` anchor with `\z`, which matches the end of the input string, ignoring all line breaks (`-z` is still necessary to avoid reading input line by line).

```
# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 6.8 (Santiago)
# echo -e "rlGetTestState\nrlRun 'echo'" | grep -Pzq 'rlGetTestState[[:space:]]*\z'; echo $?
1
# echo -e "rlGetTestState\n\n   \t\t  \n" | grep -Pzq 'rlGetTestState[[:space:]]*\z'; echo $?
0
```

Same results apply for RHEL/CentOS 7.3 and Fedora 25.

Thanks.